com.unity.netcode.gameobjects/Runtime/Interest/InterestNodeStatic.cs

```diff
@@ -0,0 +1,52 @@
+using System.Collections.Generic;
+using Unity.Netcode;
+using UnityEngine;
+
+namespace Unity.Netcode.Interest
+{
+    [CreateAssetMenu(fileName = "StaticInterestNode", menuName = "Interest/Nodes/Static", order = 1)]
+    public class InterestNodeStatic : InterestNode
```

Comment:
Interesting that the file name and the class differ. Personally would recommend this should be called StaticInterestNode (most-specific-first naming)